FeedAPIChallenge/Feed API/RemoteFeedLoader.swift

```diff
@@ -19,6 +19,66 @@ public final class RemoteFeedLoader: FeedLoader {
 	}
 	
 	public func load(completion: @escaping (FeedLoader.Result) -> Void) {
+		client.get(from: url) { [weak self] result in
+			
+			guard let _ = self else { return }
```

Comment:
Since `self` is not used within the closure, `let` is not necessary.

It's simpler to check if it's no nil:

```suggestion
			guard self != nil else { return }
```